bin/inx-chronicle/src/api/mod.rs

```diff
@@ -41,18 +46,18 @@ pub type ApiResult<T> = Result<T, ApiError>;
 #[derive(Debug)]
 pub struct ApiWorker {
     db: MongoDb,
-    config: ApiConfig,
+    config: ApiData,
```

Comment:
Should we rename the field to `api_data` in order to avoid confusion with `ApiConfig` or maybe rename `ApiData` to `ApiConfigData`?